code/modules/mob/living/simple_animal/hostile/hivebot.dm

```diff
@@ -75,24 +75,27 @@
 	icon_state = "hivebotbomber"
 	attacktext = "bumped"
 	move_to_delay = 8
+	var/has_exploded
 
 /mob/living/simple_animal/hostile/hivebot/bomber/AttackingTarget()
 	..()
 	LoseTarget()
 	stance = HOSTILE_STANCE_TIRED
 	stop_automated_movement = 1
 	wander = 0
-
-	playsound(src.loc, 'sound/items/countdown.ogg', 125, 1)
-	spawn(20)
-		fragem(src,10,30,2,3,5,1,0)
-		src.gib()
+	if(!has_exploded)
+		playsound(src.loc, 'sound/items/countdown.ogg', 125, 1)
+		has_exploded = 1
+		spawn(20)

```

Comment:
should be a timer imho but i understand if its out of scope

Reply:
Replaced it with a callback timer